BuildResidentialHPXML/measure.rb

```diff
@@ -3387,6 +3392,14 @@ def self.set_header(hpxml, runner, args)
     hpxml.header.xml_generated_by = 'BuildResidentialHPXML'
     hpxml.header.transaction = 'create'
 
+    if args[:schedules_filepaths].is_initialized
+      hpxml.header.schedules_filepaths = []
+      schedules_filepaths = args[:schedules_filepaths].get.split(',').map(&:strip)
+      schedules_filepaths.each do |schedules_filepath|
+        hpxml.header.schedules_filepaths << schedules_filepath
+      end
```

Comment:
@joseph-robertson Can't this be simply `hpxml.header.schedules_filepaths = args[:schedules_filepaths].get.split(',').map(&:strip)`?